instrumentation/opentelemetry_req/lib/opentelemetry_req.ex

```diff
@@ -115,19 +114,16 @@ defmodule OpentelemetryReq do
     {request, exception}
   end
 
-  defp format_exception(%{__exception__: true} = exception) do
-    Exception.message(exception)
-  end
-
-  defp format_exception(_), do: ""
+  defp format_error(exception) when is_exception(exception), do: Exception.message(exception)
+  defp format_error(error), do: inspect(error)
 
   defp span_name(request) do
     case request.options[:span_name] do
       nil ->
         method = http_method(request.method)
 
         case Req.Request.get_private(request, :path_params_template) do
-          nil -> method
+          nil -> "#{method} #{request.url.path}"
```

Comment:
We aren't doing 1.20+ spec piecemeal, so please keep this as it was. All the libs will have to be updated at one time.

The template on line 127 was prematurely merged and not reverted yet, so I can't publish until both are reverted to the behavior that was present prior to that PR.

Reply:
The line in question is from https://github.com/open-telemetry/opentelemetry-erlang-contrib/commit/19a44fbd6830ae71ab161028d67ce6ae40091766

Reply:
I've forgotten the plan for 1.20, hehe. Did we write it down somewhere. I vaguely remember. Maybe we should do it "now" if there are ppl looking to contribute towards the goal (not saying this should be in this PR but maybe @andrewhr can open a separate one).

Reply:
I'd like to do it all at once with 1.20 semconv. What do you think?

Reply:
#197

Reply:
From my understanding of older versions, the "best effort" of span names were always there, so this doesn't not necessary mark a breaking change from the convention definitions. In other words, was not my intend here to piecemeal implement newer specs. Having said that, I totally understand if we prefer to rollback this change - it just a matter of ergonomics so Spans can be consistent in cases we can assert they aren't dynamic.

Btw, thanks for the write up on #197, that's quite helpful! 🙇